cli.js

```diff
@@ -130,6 +131,7 @@ var api = new Api({
 	serial: cli.flags.serial,
 	require: arrify(cli.flags.require),
 	cacheEnabled: cli.flags.cache !== false,
+	disablePowerAssert: cli.flags.powerAssert === false,
```

Comment:
Double-negatives are usually bad, better to make it `powerAssert: cli.flags.powerAssert !== false,`